test/document_test.rb

```diff
@@ -290,6 +291,12 @@ def setup
     end
   end
 
+  test 'set updated at' do
+    foo = FooBar.new 'id' => 'a', foo: 'ab'
+    foo.update_properties! foo: 'c'
+    assert_equal DateTime.now.to_s, foo.updated_at.to_s
```

Comment:
I originally ommitted the `#to_s`, however when I did I recieved this

> FAIL (0:00:00.140) test_set_updated_at
          No visible difference in the DateTime#inspect output.
          You should look at the implementation of #== on DateTime or its members.